packages/spectacle/src/components/command-bar/results/index.tsx

```diff
@@ -0,0 +1,85 @@
+import styled from 'styled-components';
+import { ActionImpl, KBarResults, useMatches } from 'kbar';
+import { prettifyShortcut } from '../../../utils/platform-keys';
+import {
+  KeyboardShortcutTypes,
+  KEYBOARD_SHORTCUTS
+} from '../../../utils/constants';
+import { Text } from '../../typography';
+
+type RenderParams = {
+  item: ActionImpl | string;
+  active: boolean;
+};
+
+function getShortcutKeys({ id, shortcut = [] }: ActionImpl): string[] {
+  if (id in KEYBOARD_SHORTCUTS && !shortcut?.length) {
+    const _id = id as KeyboardShortcutTypes;
+    return prettifyShortcut(KEYBOARD_SHORTCUTS[_id]);
+  }
+  return prettifyShortcut(shortcut);
+}
+
+const ResultCommand = styled.div<Partial<RenderParams>>`
+    display: flex;
+    justify-content: space-between;
+    align-items: center
+    background-color: ${(p) => (p.active ? 'lightsteelblue' : 'transparent')};
+    padding: 0.5rem 1rem;
+    cursor: pointer;
+  `;
+
+const ResultSectionHeader = styled.div`
+  background-color: white;
+  padding-left: 1rem;
+  border-bottom: 1px solid rgba(0 0 0 / 0.1);
+  font-size: smaller;
+  margin: 0 1rem;
+`;
```

Comment:
```suggestion
const ResultSectionHeader = styled.div`
  background-color: white;
  font-size: 0.95rem;
  font-weight: 600;
  color: gray;
  margin: 0 1rem;
  padding: 0.5rem 0;
`;
```

I'm wondering if we ought to make the section headers a bit less subtle, something like the following:

![image](https://user-images.githubusercontent.com/12721310/181533347-1f3a34f5-a9ad-4d58-84dd-dfbdc8bc75f2.png)

Reply:
Sounds good to me 😅 - Im no design expert lol